main.go

```diff
@@ -22,6 +22,9 @@ import (
 // Log
 var errorLogger = log.New(os.Stderr, "ERROR ", log.Llongfile)
 
+// static variable
+var sample = 1000
```

Comment:
Given this is truly static and defined in the global scope, consider `static sample = 1000` to help out the compiler.